intro-js.md

```diff
@@ -38,6 +38,13 @@ Originally they were implemented by creating multiple functions.
 
 Classes have now been provided as part of [ECMAScript 2015](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), but this is just syntactic sugar. 
 
+#### Side Note
+What exactly is syntactic sugar? 
+
+Syntactic sugar is basically syntax for a programming language that makes it easier for developers to read and write. 
```

Comment:
Okay, I've added more to it but I'm not sure if its just going to confuse them more or not. I feel like we might be throwing too much at them with the ECMAScript standards, syntactic sugar and tipping them off to how classes are actually defined in JavaScript. That's a whole lesson in and of itself. 
Maybe it might be best to just show them how to define a class in ES6 and leave it at that.